src/sync/SyncService.ts

```diff
@@ -9,7 +9,7 @@ export enum SyncStatus {
     DATA
 }
 
-var syncServiceStatus = new ServiceStatus(defaultPingService);
+export var syncServiceStatus = new ServiceStatus(defaultPingService);
```

Comment:
why are we exporting this?

Reply:
Because I am using it in ObserverSpec.ts

Reply:
this is already exported see bottom of this file.

Reply:
also i don't see it being used in that file i checked